h2o-automl/src/test/java/hex/ModelBuilderAutoMLTest.java

```diff
@@ -0,0 +1,141 @@
+package hex;
```

Comment:
does it make sense to have this test suite in AutoML module?

Reply:
@seb-h2o I also don't like this. That is why I left this comment from the beginning https://github.com/h2oai/h2o-3/pull/3911/files#r327223191 . It's complicated to write test in h2o-core module ( see answer https://github.com/h2oai/h2o-3/pull/3911#discussion_r328227400). And for me it is better to have test in a not the best place than don't have it at all. I will try to do something about it, maybe @michalkurka will suggest me something.